vignettes/01_rphenoscape.Rmd

```diff
@@ -51,12 +51,12 @@ The [RPhenoscape] package provides convenient access to the Phenoscape Knowledge
 library(rphenoscape)
 ```
 
-First let’s look up information about the query terms. You can use the term detail functions ```pk_anatomical_detail()```, ```pk_taxon_detail()```, ```pk_phenotype_detail()```, and ```pk_gene_detail()``` to retrieve an ontology term’s ID, label, and definition. 
+First let’s look up information about the query terms. You can use the term detail functions ```anatomy_term_info()```, ```taxon_info()```, ```phenotypic_quality_term_info()```, and ```gene_info()``` to retrieve an ontology term’s ID, label, and definition. 
```

Comment:
I would not advertise the use of the `XXXX_info()` functions. The streamlined way to do this is using `as.terminfo()`.

Reply:
BTW I'm also wondering why the triple backtick was used for inline code quoting. It should be a single backtick.

Reply:
done, please review my recent update on this branch

Reply:
Not sure about the triple backtick. I changed these to single.